src/store/metadata/stub/content/versions.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
you'll need to add "in_dataset" and "in_edition" fields to go with your store.get_versions() holding logic.

ideally, we want to implement as far as we can, so building on the existing example we'll want to be hitting `/dataets/cpid/editions/2022-01` to trigger logic to return the contents of this file.

so in_dataset = "cpih" and in_edition="2022-01"

I'd also add one more mocked up item with a slightly different in_edition value just to make sure we're _only_ returning what's asked for.